### PR TITLE
Designed several webpages

### PR DESCRIPTION
**Pull request by pnoorossana**
Updated the event and seating chart pages.

### CODE REVIEW

File: eagleEvents/templates/event.html.j2 [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% extends "layout.html.j2" %}
+{% block head_content %}
+    <title>Eagle Events Event</title>
+{% endblock %}
+    <h1>Events</h1>
```
Comment (simpleman19) [inline eagleEvents/templates/event.html.j2]: Missing a start block for the body content block

File: eagleEvents/templates/seating-chart.html.j2 [hunk 2]
```diff
@@ -0,0 +1,211 @@
+{% extends "layout.html.j2" %}
+{% block head_content %}
+      <title>Eagle Events Seating Chart</title>
+{% endblock %}  
+{% block body_content %}
+    <nav class="navbar navbar-default">
+        <div class="container-fluid">
```
Comment (simpleman19) [inline eagleEvents/templates/seating-chart.html.j2]: I don't think this page should have a nav bar, that should be taken care of by the layout page

File: eagleEvents/templates/event.html.j2 (hunk 1 above)
Comment (pnoorossana) [reply eagleEvents/templates/event.html.j2]: Added a start block for the body content block to the event page and also fixed the table header.

File: eagleEvents/templates/seating-chart.html.j2 (hunk 2 above)
Comment (pnoorossana) [reply eagleEvents/templates/seating-chart.html.j2]: Removed the Navbar from the page.

**Pull request merged**